modules/030-cloud-provider-yandex/templates/cloud-metrics-exporter/deployment.yaml

```
@@ -0,0 +1,111 @@
+{{- if .Values.cloudProviderYandex.cloudMetricsExporterEnabled }}
+{{- if (.Values.global.enabledModules | has "vertical-pod-autoscaler-crd") }}
+---
+apiVersion: autoscaling.k8s.io/v1
+kind: VerticalPodAutoscaler
+metadata:
+  name: cloud-metrics-exporter
+  namespace: d8-cloud-provider-yandex
+  {{- include "helm_lib_module_labels" (list . (dict "app" "cloud-metrics-exporter")) | nindent 2 }}
+spec:
+  targetRef:
+    apiVersion: "apps/v1"
+    kind: Deployment
+    name: cloud-metrics-exporter
+  updatePolicy:
+    updateMode: "Initial"
+{{- end }}
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: cloud-metrics-exporter
+  namespace: d8-cloud-provider-yandex
+  {{- include "helm_lib_module_labels" (list . (dict "app" "cloud-metrics-exporter")) | nindent 2 }}
+spec:
+  {{- include "helm_lib_deployment_strategy_and_replicas_for_ha" . | nindent 2 }}
+  selector:
+    matchLabels:
+      app: cloud-metrics-exporter
+  template:
+    metadata:
+      labels:
+        app: cloud-metrics-exporter
+      annotations:
+        checksum/config: {{ include (print $.Template.BasePath "/cloud-metrics-exporter/secret-exporter-creds.yaml") . | sha256sum }}
+    spec:
+      serviceAccountName: cloud-metrics-exporter
+      imagePullSecrets:
+      - name: deckhouse-registry
+      {{- include "helm_lib_node_selector" (tuple . "monitoring") | nindent 6 }}
+      {{- include "helm_lib_tolerations" (tuple . "monitoring") | nindent 6 }}
+      {{- include "helm_lib_priority_class" (tuple . "system-cluster-critical") | nindent 6 }}
```

```suggestion
      {{- include "helm_lib_priority_class" (tuple . "cluster-low") | nindent 6 }}
```
Should not be higher than the Prometheus priority class.